### PR TITLE
MISUV-475: Combine models, keystore, cache utils for individual and a…

### PR DESCRIPTION
**Pull request by scullythornton1**
…gent journeys

### CODE REVIEW

File: test/controllers/utils/RequirementsSpec.scala [hunk 1]
```diff
@@ -117,21 +116,10 @@ class RequirementsSpec extends WordSpecLike with MustMatchers with MockitoSugar
   }
 
   "require" must {
-    "redirect to the no data redirect location" when {
```
Comment (chandraAndem) [inline test/controllers/utils/RequirementsSpec.scala:120]: Looks like a valid test case, any specific reason to remove this?

Comment (scullythornton1) [reply test/controllers/utils/RequirementsSpec.scala:120]: before fetchAll function for the agent is a type of [Option], after combining the individual and agent KeyStoreServices, the remove the fetchAll for the agent and kept the fetchAll for the individual which is returning the type Future[CacheMap], thus there will not be a scenario when return "No data".

File: app/controllers/individual/subscription/ConfirmationController.scala [hunk 2]
```diff
@@ -18,23 +18,21 @@ package controllers.individual.subscription
...
-import utilities.individual.CacheUtil._
+import utilities.CacheUtil._
 import views.html.individual.incometax.subscription.sign_up_complete
 
 import scala.concurrent.ExecutionContext
 
 @Singleton
 class ConfirmationController @Inject()(val authService: AuthService, keystoreService: KeystoreService)(implicit val ec: ExecutionContext, appConfig: AppConfig,
-                                       mcc: MessagesControllerComponents) extends PostSubmissionController {
+                                                                                                       mcc: MessagesControllerComponents) extends PostSubmissionController {
```
[the remark this answers is not in the document]
Comment (scullythornton1) [reply app/controllers/individual/subscription/ConfirmationController.scala:35]: amended. :)

File: test/controllers/utils/RequirementsSpec.scala (hunk 1 above)
Comment (chandraAndem) [reply test/controllers/utils/RequirementsSpec.scala:120]: Thanks!

**Pull request merged**